CHANGELOG.md

```diff
@@ -1,5 +1,78 @@
 
 
+## [2.4.4](https://github.com/chargebee/chargebee-react-native/compare/v2.4.3...v2.4.4) (2023-11-10)
+
+## [2.4.3](https://github.com/chargebee/chargebee-react-native/compare/v2.4.2...v2.4.3) (2023-10-09)
+
+## [2.4.2](https://github.com/chargebee/chargebee-react-native/compare/v2.4.1...v2.4.2) (2023-09-01)
+
+
+### Bug Fixes
+
```

Comment:
@cb-haripriyan Just curious why our change log starts with small letters. I checked a few other open-source products they seem to follow normal English

Reply:
The change logs are autogenerated from the commit messages, using [release-it](https://github.com/release-it/release-it/tree/main#changelog).
We can modify the template as we need.